compiler/x/codegen/X86Ops.ins

```diff
@@ -615,6 +615,15 @@ INSTRUCTION(VBROADCASTSDZmmXmm, vbroadcastsd,
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM),
             PROPERTY1(IA32OpProp1_XMMSource | IA32OpProp1_XMMTarget | IA32OpProp1_SIMDSingleSource),
             FEATURES(X86FeatureProp_EVEX512Supported | X86FeatureProp_EVEX512RequiresAVX512F)),
+INSTRUCTION(VPBROADCASTBRegReg, vpbroadcastb,
+            BINARY(VEX_L128, VEX_vNONE, PREFIX_66, REX__, ESCAPE_0F38, 0x78, 0, ModRM_RM__, Immediate_0),
```

Comment:
You should add some binary encoding tests in [BinaryEncoder.cpp](https://github.com/eclipse-omr/omr/blob/3b5210dbcb49de05e04d3f7ef9a9f4131428ed28/fvtest/compilerunittest/x/BinaryEncoder.cpp#L380-L481).